### PR TITLE
W-9118085 Share Button Available on Review, Funding Request and Requirements Object

### PR DESCRIPTION
**Pull request by seanpat09**
Screenshot of how the new Sharing button displays on page layouts:

Screen Shot 2021-04-26 at 2 43 39 PM

# Critical Changes

# Changes

Adds the Sharing button to the Lightning page layouts for the Review, Funding Request, and Requirement objects.

# Issues Closed

# New Metadata

# Deleted Metadata

# Definition of Done
Refer to [Definition of Done](https://salesforce.quip.com/9P7hAOPHJJyU) to see any additional details for the items below:
- ~[ ] Any net new LWC work has JEST test coverage 50% or above~
- ~[ ] Default Sa11y tests pass for all LWC components~
- ~[ ] 🔒 Secure both Front-end (LWC) & back-end (Apex) as necessary~
- ~[ ] 🔑 Grant users access in Permission Sets (Object, Field, Apex Class) as necessary~
- [x] Link the pull request and work item by PR comment and Chatter post respectively, e.g. GUS: [W-0000000: Work Name]()
- [x] Make sure that ACs are updated (if any gaps)
- [x] **All acceptance criteria have been met**
    - [x] Developer
    - [x] Code Reviewer
- [x] Pull Request contains draft release notes
- ~[ ] Labels, help text, and customer facing messages are reviewed by Docs~
- [ ] QE story level testing completed

### CODE REVIEW

**Comment by seanpat09** [comment]
[W-9118085 Share Button Available on Review, Funding Request and Requirements Object](https://gus.lightning.force.com/a07AH000000AV4gYAG)

File: force-app/main/default/layouts/Funding_Request__c-Funding Request Layout.layout-meta.xml [hunk 1]
```diff
@@ -310,7 +315,7 @@
     <showRunAssignmentRulesCheckbox>false</showRunAssignmentRulesCheckbox>
     <showSubmitAndAttachButton>false</showSubmitAndAttachButton>
     <summaryLayout>
-        <masterLabel>00h2D000002AEXN</masterLabel>
+        <masterLabel>00h2D000001yQY1</masterLabel>
```
Comment (seanpat09) [inline force-app/main/default/layouts/Funding_Request__c-Funding Request Layout.layout-meta.xml:318]:
```suggestion
        <masterLabel>\00h2D000002AEXN</masterLabel>
```

File: force-app/main/default/layouts/Funding_Request__c-Funding Request Layout.layout-meta.xml [hunk 2]
```diff
@@ -310,7 +315,7 @@
     <showRunAssignmentRulesCheckbox>false</showRunAssignmentRulesCheckbox>
     <showSubmitAndAttachButton>false</showSubmitAndAttachButton>
     <summaryLayout>
-        <masterLabel>00h2D000002AEXN</masterLabel>
+        <masterLabel>\00h2D000002AEXN</masterLabel>
```
Comment (seanpat09) [inline force-app/main/default/layouts/Funding_Request__c-Funding Request Layout.layout-meta.xml:318]:
```suggestion
        <masterLabel>00h2D000002AEXN</masterLabel>
```

File: force-app/main/default/layouts/Requirement__c-Requirement Layout.layout-meta.xml [hunk 3]
```diff
@@ -139,14 +139,14 @@
             <sortOrder>4</sortOrder>
         </platformActionListItems>
         <platformActionListItems>
-            <actionName>FeedItem.TextPost</actionName>
+            <actionName>SendEmail</actionName>
```
Comment (seanpat09) [inline force-app/main/default/layouts/Requirement__c-Requirement Layout.layout-meta.xml:142]: This change looks like the buttons got reordered, but it `FeedItem.TextPost` and `SendEmail` are actually still in the same order as before. The metadata just had `<sortOrder>6</sortOrder>` listed before `<sortOrder>5</sortOrder>` previously

**Review by jdavilasf — APPROVED** [review]
The changes look good. @good job @seanpat09 👍

**Pull request merged**